src/tests/ftest/nvme/nvme_object.yaml

```diff
@@ -29,20 +29,24 @@ agent_config:
 dmg:
   transport_config:
     allow_insecure: True
-pool:
-  createmode:
-    mode: 511
-  createset:
-    group: daos_server
-  createsize:
-    scm_size: 4000000000
-    size:
-      - 20000000000
-      - 100000000000
-      - 800000000000
-  createsvc:
-    svcn: 1
+pool_1:
   control_method: dmg
+  name: daos_server
+  scm_size: 4000000000
+  nvme_size: 20000000000
```

Comment:
I wish we'd use human-readable values in these files.  I have a PR that moves all of these values to human-readable values in all yaml files.  Most assuredly bit-rotten by now.  :frowning:

Reply:
I don't think human-readable values were supported when some of this test was developed. That is the reason many of the test codes still use the plain size including rebuild/container/pool test folders.

Reply:
Yes, completely understood.  But that doesn't apply any more and IMHO, human readable values should be the norm going forward.

Reply:
I agree if we are changing anyway better to use a human-readable format.